x-pack/plugins/security/public/management/roles/edit_role/edit_role_page.tsx

```diff
@@ -718,12 +718,21 @@ export const EditRolePage: FunctionComponent<Props> = ({
           ),
         });
       } else {
-        notifications.toasts.addSuccess(
-          i18n.translate(
-            'xpack.security.management.editRole.roleSuccessfullySavedNotificationMessage',
-            { defaultMessage: 'Saved role' }
-          )
-        );
+        if (isEditingExistingRole) {
+          notifications.toasts.addSuccess(
+            i18n.translate(
+              'xpack.security.management.editRole.roleSuccessfullyUpdatedNotificationMessage',
+              { defaultMessage: 'Updated role' }
+            )
+          );
+        } else {
+          notifications.toasts.addSuccess(
+            i18n.translate(
+              'xpack.security.management.editRole.roleSuccessfullySavedNotificationMessage',
```

Comment:
I should have highlighted the i18n path last time as well

```suggestion
              'xpack.security.management.editRole.roleSuccessfullyCreatedNotificationMessage',
```